src/components/User-Landing-Page/User-Landing.js

```diff
@@ -0,0 +1,71 @@
+import React, { useState, useEffect } from 'react'
+
+function UserLandingPage() {
+  const [user, setUser] = useState({id: 1, name: 'John', address: '123 Wherever St, Denver CO 80202', connections: false, about: 'My about me'})
+  const [connection, setConnection] = useState({id: 2, name:'Bill', about:'Howdy Im Bill'})
+
+  useEffect(() => {
+    console.log(user.connections)
+  }, [])
+
+  // PASS USER DATA THRU AS PROPS AFTER LOGIN, OR CALL THEM ON PAGE LOAD?
+
+  return (
+    <div className='landing-wrapper'>
+      <div className='header-placeholder'/>
+      <h1>Welcome {user.name}</h1>
+      <h2>Profile Info</h2>
+      <div className='info-wrapper'>
+        <h3>Address:</h3>
+        <h4>{user.address}</h4>
+        <h3>About:</h3>
+        <h4>{user.about}</h4>
+        <button>Edit</button>
+      </div>
+      <h2>Connection</h2>
+      <div className='info-wrapper'>
+        {!connection &&
```

Comment:
is this checking if the connection object is a falsey value? Is an empty object a falsey value or is it an array?

Reply:
Good point. We could run a check by having a second, commented out `useState` for our connection set to the default of `null`. 
We might also want to confirm with the backend if a user starts with a connection or doesn't. I forget.